receiver/k8sclusterreceiver/internal/service/service.go

```diff
@@ -24,6 +24,11 @@ func Transform(service *corev1.Service) *corev1.Service {
 	}
 }
 
+// Maximum number of services assigned to a pod that will be added as properties.
+// If a pod has more services than this limit, it's typically an ingress controller or a similar service.
+// In that case, we don't set any service properties to the pod since it's not useful.
```

Comment:
I wonder if this assumption should be publicly documented for the users to avoid any possible confusion.

Reply:
I don't believe this experimental functionality is properly documented at this point